backend/cn/lib/executable_spec_utils.ml

```diff
@@ -155,3 +155,36 @@ let create_decl_object ctype =
 
 (* declarations: list (ail_identifier * (Loc.t * Annot.attributes * declaration)) *)
 let create_declaration sym decl = (sym, (Cerb_location.unknown, CF.Annot.Attrs [], decl))
+
+let get_start_loc ?(offset = 0) = function
```

Comment:
a bit different to `Locations.start_pos`, because of the offset optional parameter. but I can rewrite them to use the `Locations` functions (and I found a mistake in my implementation by looking at those functions, so thank you)